mesonbuild/build.py

```diff
@@ -377,11 +377,24 @@ class IncludeDirs(HoldableObject):
     # Interpreter has validated that all given directories
     # actually exist.
     extra_build_dirs: T.List[str] = field(default_factory=list)
+    build_relative: bool = field(default=True)
+    source_relative: bool = field(default=True)
```

Comment:
```suggestion
    build_relative: bool = True
    source_relative: bool = True
```

While not wrong, you don't really need `field` here, since you only provide the default value